transports/sftp/src/test/resources/sftp-proxy-config.xml

```diff
@@ -0,0 +1,26 @@
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:spring="http://www.springframework.org/schema/beans"
+    xmlns:sftp="http://www.mulesoft.org/schema/mule/sftp"
+    xmlns:vm="http://www.mulesoft.org/schema/mule/vm"
+    xsi:schemaLocation="
+        http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-current.xsd
+        http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+        http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd
+        http://www.mulesoft.org/schema/mule/sftp http://www.mulesoft.org/schema/mule/sftp/current/mule-sftp.xsd">
+
+    <spring:beans>
+        <spring:import resource="sftp-properties.xml" />
+    </spring:beans>
+
+    <sftp:connector name="proxyConnector">
+        <sftp:proxy-config host="localhost" port="${proxyPort}"/>
+    </sftp:connector>
+
+    <flow name="sftpThroughProxy">
+        <!-- Use an invalid address so that it will fail if it doesn't go through the proxy -->
+        <sftp:inbound-endpoint
+            address="sftp://${USER1_NAME}:${USER1_PASSWORD}@127.0.0.100:${SFTP_PORT}/~/testdir"
```

Comment:
No, the address is the real one. The proxy is just the middle man connecting to the address.
It's probably confusing because the test proxy ignores the CONNECT argument, but a real proxy connects to whatever address one tells it to do.